src/pointcloud_io.jl

```diff
@@ -403,6 +428,37 @@ function make_table(points::AbstractVector{LasPoint7}, offset, scale)
                  color = color)
 end
 
+function make_table(points::AbstractVector{LasPoint8}, offset, scale)
+    position = map(points) do p
+        @inbounds SVector(p.x*scale[1] + offset[1], p.y*scale[2] + offset[2], p.z*scale[3] + offset[3])
+    end
+
+    intensity = map(p -> p.intensity, points)
+    returnnumber = map(p -> p.flag_byte_1 & 0b00001111, points)
+    numberofreturns = map(p -> (p.flag_byte_1 & 0b11110000) >> 4, points)
+    classification = map(p -> p.classification, points)
+    pointsourceid = map(p -> p.pt_src_id, points)
+    gpstime = map(p -> p.gps_time, points)
+    userdata = map(p -> p.user_data, points)
+    nir = map(p -> p.nir, points)
```

Comment:
did you test this on a point format 8 file? if not that's fine just asking

Reply:
yes!